gunicorn/util.py

```diff
@@ -364,19 +365,18 @@ def import_app(module):
         module, obj = parts[0], parts[1]
 
     try:
-        __import__(module)
+        mod = importlib.import_module(module)
     except ImportError:
+        # ModuleNotFoundError (a subclass of ImportError) is new in version 3.6.
```

Comment:
This comment can be dropped.

Reply:
I just wanted to tell `ModuleNotFound` is handled by this clause. But It's ok to remove this comment.

Reply:
It's probably not necessary to have the comment. There's only one statement in the `try` clause, so even if the error is more general than needed on Python 3.6 there's not much need to justify it.

Reply:
I see. I've just removed this line.